packages/tasit-action/package.json

```diff
@@ -17,7 +17,7 @@
     "url": "git+https://github.com/tasitlabs/tasitsdk.git"
   },
   "scripts": {
-    "prepare": "npx babel src --out-dir dist --ignore **/*.test.js",
+    "prepare": "npx babel src --out-dir dist --ignore **/*.test.js && cp src/abi/ dist/ -r",
```

Comment:
I've used NFT class on one of my tryouts, and It didn't work because the the the `dist/abi/ERC721Full.json` file doesn't exist.

Reply:
Ah ok - makes sense. Would it make more sense to copy it to `dist/abi/` instead of `dist/`?

Reply:
It's copying the `abi/` dir to inside of `dist/`
If I change to `cp src/abi/ dist/abi/ -r` it will copy `src/abi` to inside of `dist/abi` if it already exists.

Reply:
Right now it's copying the dir itself or the contents of the dir into `dist/`?

Reply:
Basically I think it will be less confusing if we have the `dist` dir mirror the `src` dir as much as possible. But it may already be working that way.

Reply:
1) It is copying the dir with their contents.
2) Maybe if we have something like that:
`"prepare": "rm -rf dist/* && npx babel src --out-dir dist --ignore **/*.test.js && mkdir dist/abi && cp src/abi/* dist/abi/"`
It's be more clearly code.

Reply:
Yeah, that looks more readable to me. Thanks!

I don't think the dist clearing will add too much time to the tests - do you?

Reply:
I agree, will don't.

Reply:
Done

Reply:
Thanks. Resolving.